package.json

```diff
@@ -19,7 +19,7 @@
     "test:api": "run-script-os",
     "test:api:win32": "npx codeceptjs run-workers --suites 4 --grep @api-tests --reporter mocha-multi --verbose",
     "test:api:darwin:linux": "MOCHAWESOME_REPORTFILENAME=api npx codeceptjs run-workers --suites 4 --grep @api-tests --reporter mocha-multi --verbose",
-    "test:e2e": "MOCHAWESOME_REPORTFILENAME=e2e npx codeceptjs run-workers --suites 2 --grep @e2e-tests --reporter mocha-multi --verbose",
+    "test:e2e": "MOCHAWESOME_REPORTFILENAME=e2e npx codeceptjs run --grep @e2e-tests --reporter mocha-multi --verbose",
```

Comment:
is this on purpose ?

Reply:
yes @vasudevganesanhmcts this script is for local only hence i removed workers